onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -3672,6 +3672,10 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
         return ORT_MAKE_STATUS(ONNXRUNTIME, EP_FAIL, "TensorRT EP Failed to Build Engine.");
       }
       trt_engine = trt_state->engine->get();
+      // Check before using trt_engine
+      if (trt_engine == nullptr) {
```

Comment:
No need to check null pointer here since it's identical to line 3671 above.

Reply:
Fixed, thanks!